uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Theme.kt

```diff
@@ -67,3 +94,9 @@ fun AppThemeWithBackground(
         ConferenceAppFeederTheme(theme, content)
     }
 }
+
+object ConferenceAppFeederTheme {
+    val filterMuskColor: Color
+        @Composable
+        get() = LocalFilterMuskColor.current
```

Comment:
enable to use LocalFilterMuskColor without publishing this object outside of core module.
I referred compose-samples code to do it.